std_msgs/Header header
uint32 destination_robot_id
BowQuery queries